app/Resources/views/base.html.twig

```diff
@@ -154,6 +154,15 @@
             <script>
                 $(document).ready(function() {
                     hljs.initHighlightingOnLoad();
+
+                    $('form[data-confirmation-message]').on('submit', function (event) {
```

Comment:
stof sorry, I did it on purpose, you could tell me how to apply event delegation here? 

something like this?:
```javascript
$('body').delegate('form[data-confirmation-message]', 'submit', function (e) {...});
```